src/hist/basehist.py

```diff
@@ -5,11 +5,15 @@
 import matplotlib.pyplot as plt
 import matplotlib.patches as patches
 from matplotlib import transforms
+import warnings
 from scipy.optimize import curve_fit
 from uncertainties import correlated_values, unumpy
 from typing import Callable, Optional, Tuple, Union, List, Any
 
 import hist.utils
+import hist.storage
+from hist.storage import Storage
```

Comment:
It's nice to leave full names for things unless they are common or part of a complex expression, or are Types.